pkg/ceph/ceph.go

```diff
@@ -0,0 +1,57 @@
+package cephcommands
+
+import (
```

Comment:
As much as possible, I would hope that we can avoid implementation under `pkg/` since we expect the implementation to be in the upstream tool. The `odf` tool hopefully is just a thin `cmd/` wrapper around the upstream tool, at least for the rook and ceph commands.